cpp/src/slice2matlab/Main.cpp

```diff
@@ -314,7 +50,20 @@ namespace
 
     void openClass(const string& abs, const string& dir, IceInternal::Output& out)
     {
```

Comment:
This was the only place left where we called `splitAbsoluteName`, so I just inlined the logic.